### ISSUE TITLE
Support native RSpec example filtering

### ISSUE THREAD

**Issue by ianks** [skroutz/rspecq#23]
For test performance, we split our suite into `acceptance` and `unit` tests. We do this because `acceptance` spec require static assets to be built, which takes a decent amount of time which, whereas `unit` specs can be run instantly. We accomplish this with RSpec tabs. For example, here is how we run our `acceptance` tests using rspec-queue:

```sh
            bin/_rspec-queue \
              --namespace acceptance \
              --tag capybara_feature \
              --tag type:feature \
              --tag type:request \
              --tag js \
              --tag webpack \
              --format=doc \
              --format=RspecJunitFormatter \
              --out="tmp/test_results/rspec_acceptance/results-$CIRCLE_NODE_INDEX.xml" \
              --requeue-tolerance=0.05 \
              --max-requeues="$CIRCLE_NODE_TOTAL"
```

In order to use rspecq, we need to mimic this functionality. As of now, it does not seem like rspecq supports example filtering, is this right? If not, would it be possible to support this use case?

**Comment by agis** [comment]
Right, currently we don't support passing tags, simply because we haven't encountered the need to. It's certainly a valid feature though. I guess it would involve somehow passing the tags argument in here, similar to how we pass `--format`: https://github.com/skroutz/rspecq/blob/e42cae40978b132eb5e5ea31640a6711c8055961/lib/rspecq/worker.rb#L89

The CLI should also be extended to support this argument.
